Sending messages sometimes results in a ClientStatusException

**Description**

Now that we fail builds on flaky tests, periodically the build will fail on tests where we send a message using the `ZeebeClient`.
Sending the message will result in a timeout.

```
io.camunda.zeebe.client.api.command.ClientStatusException: deadline exceeded after 9.999761982s. [closed=[], open=[[remote_addr=0.0.0.0/0:0:0:0:0:0:0:1:1123]]]
	at io.camunda.zeebe.bpmnassert.assertions.ProcessInstanceAssertTest$HappyPathTests.testHasCorrelatedMessageByCorrelationKey(ProcessInstanceAssertTest.java:403)
Caused by: java.util.concurrent.ExecutionException: io.grpc.StatusRuntimeException: DEADLINE_EXCEEDED: deadline exceeded after 9.999761982s. [closed=[], open=[[remote_addr=0.0.0.0/0:0:0:0:0:0:0:1:1123]]]
	at io.camunda.zeebe.bpmnassert.assertions.ProcessInstanceAssertTest$HappyPathTests.testHasCorrelatedMessageByCorrelationKey(ProcessInstanceAssertTest.java:403)
Caused by: io.grpc.StatusRuntimeException: DEADLINE_EXCEEDED: deadline exceeded after 9.999761982s. [closed=[], open=[[remote_addr=0.0.0.0/0:0:0:0:0:0:0:1:1123]]]
```

--- a/.github/workflows/build-test.yml
+++ b/.github/workflows/build-test.yml
@@ -8,6 +8,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
+      max-parallel: 1
       matrix:
         os: [ macos-latest, windows-latest, ubuntu-latest ]
 

